src/main/java/domain/WinningResult.java

```diff
@@ -8,21 +8,15 @@ public class WinningResult {
 	private final Map<Rank, Integer> winningResult;
 	private final WinningMoney winningMoney;
```

Comment:
인스턴스 변수를 줄여보면 어떨까 싶어요. 😃
WinningResult가 `winningMoney` 상태 값을 꼭 가져야할까요? 🤔

왜 WinningResult는 `winningResult` 뿐만아니라 `winningMoney`를 상태 값으로 가져야할까요? 

```suggestion
	private final Map<Rank, Integer> winningResult;
```

WinningResult는 단순히 `WinningMoney`를 계산해주는 역할만 가져가면 어떨까요?

Reply:
말씀하신대로 상태 없앴고, `WinningResult#getWinningMoney`에서 원래 가지고있던 `winningMoney`를 계산하여 반환하는 로직 추가했습니다. `add()`부분이 날씬해져서 좋네요.